src/gui/src/gui.tcl

```diff
@@ -181,6 +181,47 @@ proc select { args } {
   return [gui::select $type $name $case_sense $highlight]
 }
 
+sta::define_cmd_args "timing_cone" {iterm_or_bterm \
```

Comment:
this should be named something like display_timing_cone because it does not find/return a timing code.
command args should be consistent with sdc (where there is no such thing as iterm or bterm) but I haven't
had much luck enforcing this obvious rule.
if this is just for testing it should use define_hidden_cmd_args

Reply:
It is public and I agree with your suggested name.

Reply:
renamed and switched to use sta pin as tcl input (similar to all the other sta commands)